Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cargo_metadata"
-version = "0.18.0"
+version = "0.19.0"
```

Comment:
Ah, since we are pre-1.0, a minor bump is a breaking bump. We can use a patch bump here